pkg/distributor/http_server.go

```diff
@@ -92,17 +92,20 @@ func (d *Distributor) ValidateExprHandler(w http.ResponseWriter, r *http.Request
 	// and the generated error string omits the line entirely. But we
 	// want to report line numbers consistently, no matter how many
 	// lines there are (starting at 1).
-	if parseErr.Line == 0 {
-		parseErr.Line = 1
-	}
+	// Not available from Prometheus at the moment.
```

Comment:
I think it's fine to remove the commented out code entirely here. I don't think we will bring it back any time soon in this case.